tests/integration/operator/uninstall_test.go

```diff
@@ -109,72 +109,6 @@ func TestReconcileDelete(t *testing.T) {
 					return nil
 				}, retry.Timeout(deletionTimeout), retry.Delay(retryDelay))
 			})
-
-			// For negative casse, use default profile, ingressgateway alway connect to istiod
-			// deletion will fail
-			t.NewSubTest("delete-iop-fail").Run(func(t framework.TestContext) {
```

Comment:
Remove the erroneous test case.
My understanding is that the ingressgateway should not impede delete iop action of the IstioOperator.

Reply:
This makes sense to me, needs @zirain to confirm.

Reply:
@zirain  PTAL 😄

Reply:
this seems not right to me.
IIRC, gateway are shared across differnet revisions, so we should check there's no proxies are connected to istiod before removing.

Reply:
If there's a revision with an ingress gateway, it means users have performed the revision installation including the gateway component. Otherwise, they should have installed using a minimal profile without incorporating the gateway. If they then proceed to uninstall that revision, the gateway component should be uninstalled as well to keep the consistency.

Reply:
Regarding whether the gateway should affect user uninstallation operations, we currently have different understandings. @howardjohn  what is your perspective on this matter ❓ ❓